db_test.go

```diff
@@ -124,6 +124,27 @@ func signin(s *SurrealDBTestSuite) interface{} {
 	return signin
 }
 
+func (s *SurrealDBTestSuite) TestLive() {
+	s.Run("Live notifications", func() {
+		live, err := s.db.Live("users")
+		s.Require().NoError(err)
+		notifications, er := s.db.LiveNotifications(live.(string))
+		// create a user
+		s.Require().NoError(er)
+		_, e := s.db.Create("users", map[string]interface{}{
+			"username": "johnny",
+			"password": "123",
+		})
+		s.Require().NoError(e)
+		notification := <-notifications
+		res := notification.Result.(map[string]interface{})
+		s.Require().Equal("CREATE", res["action"])
+		s.Require().Equal(live.(string), res["id"])
+		// Need to kill before get out from scope
+		_, err = s.db.Kill(live.(string))
```

Comment:
This is awesome, nice catch!
You can call it immediately after live using `defer`.
Adding suggestion above as well. The assert can be removed from here.
```suggestion

```